domains/pizza.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "oliik2013",
+           "email": "oliik2013@oliik2013.is-a.dev",
+           "discord": "coolname4262"
+        },
+    
+        "record": {
+            "CNAME": "oliik2013.github.io"
```

Comment:
Can you show a preview of your website?

Reply:
https://oliik2013.github.io/strona-flash/